reproducibility_FAQs.md

```diff
@@ -0,0 +1,253 @@
+
+- *Incomplete Version Control or Environment Info:**  
+  Failing to specify software versions (e.g., using Stata 18 but not declaring it), or not locking Python/R environments (e.g., with `requirements.txt`, `renv.lock`) can cause code to behave differently on other systems. Reproducibility requires knowing exactly what software environment was used. [This presentation](https://osf.io/q7rjt) contains more information on this issue and potential solutions.
+
+- **Manuscript Version Not Aligned with Code:**  
+  In some cases, the code is up to date but the manuscript is not. Or vice versa—the manuscript reflects newer analyses, but the package reflects an older draft. This creates discrepancies between exhibits and outputs. Always ensure that the final manuscript matches the results generated by the package.
+
+
+### What if my package uses Excel?
+
+While Excel is widely used, reproducibility verification is  **much more difficult**  when workflows rely on manual steps, hidden formulas, or undocumented edits. Therefore,  **we strongly discourage using Excel for critical parts of the analysis**  and recommend automating as much of the workflow as possible.
+
+If your package includes Excel files:
+
+-   **Avoid manual steps**  (e.g., copy-pasting, deleting rows)
+-   **Use formulas and macros**  where appropriate
+-   **Document all transformations clearly**  — either in a separate log file or within the README
+-   **Explain how outputs were created from Excel files**, especially if they appear in the manuscript
+    
+
+
+ [See our Excel guidelines](https://worldbank.github.io/wb-reproducible-research-repository/resources/excel/Excel_Guidelines.html)  for recommended practices and our [presentation on how to produce tables and plots in Stata and R](https://osf.io/ezmsb) for quick instructions on how to build reproducible outputs.
+
+## Data-Related Questions
+
+### What is original data?
+Original data refers to a citable dataset—one that is documented, publicly accessible or archived in a World Bank internal catalog—and serves as the starting point for reproducibility.  A few options include:
+-   Unmodified raw data: As originally received, from surveys, administrative systems, or fieldwork, with no changes beyond the removal of personally identifiable information(PII).  
+-   Published datasets: Datasets from trusted sources like the Microdata Library, LSMS, DHS, etc.  
+-   Official statistics & indicators: Data from sources like the World Development Indicators, national statistics portals, or IMF/WEO
+-   Data as it was received from another WB team or an external counterpart, except for the removal of personally identifiable information (see the question below on documenting the data sources for more information).
+
+### Can you do a reproducibility verification if the research relies on confidential data?
```

Comment:
adapt references to confidential to be proprietary or confidential throughout (meanings overlap but people use both so let's be inclusive)